### PR TITLE
3.5-Fix POG Slow startup issue

### PR DESCRIPTION
**Pull request by BrentEaston**

### CODE REVIEW

**Comment by BrentEaston** [comment]
Broken test. Working on a fix.

**Comment by BrentEaston** [comment]
@Cattlesquat Could you please do a final POG check on this PR Brian? I've just shuffled a few things around, should not have made a difference, just want to be double sure. Thanks.

File: vassal-app/src/main/java/VASSAL/configure/Configurer.java [hunk 1]
```diff
@@ -149,11 +153,37 @@ public void removePropertyChangeListener(PropertyChangeListener l) {
...
+   * The headless check is required in case any Configurers try to initialize during tests.
+   *
+   * @return Screen Size.
+   */
+  protected Dimension getScreenSize() {
+    if (GraphicsEnvironment.isHeadless()) {
+      return new Dimension(1920, 1280);
+    }
+    else {
+      return Toolkit.getDefaultToolkit().getScreenSize();
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/configure/Configurer.java:186]: Good solution.

**Review by uckelman — APPROVED** [review]

**Comment by Cattlesquat** [comment]
PR runs great with PoG now

**Pull request merged**